### ISSUE TITLE
Audio Muting for TNT Drama

### ISSUE THREAD

**Issue by richardfrost** [FrostCo/AdvancedProfanityFilter#657]
### :movie_camera: [TNT Drama](https://www.tntdrama.com)

This issue is here for sharing updates and reporting issues related to this supported site. Feel free to leave a comment if you have any feedback to share.

**Comment by fishmenr320** [comment]
Audio muting is not working on this site. (www.tntdrama.com)  The subtitle text is filtered but the sound is never muted. This issue is similar to what is described in issue #677 for NOW TV. I have tried to get muting on tntdrama.com to work on both Firefox and Vivaldi browsers without success.  Also on a side note, tbs.com and trutv.com are similar/related websites and it would seem you could easily add both of these to your audio muting site list (as they may take the same settings as tntdrama).

**Comment by richardfrost** [comment]
As was reported, it appears filtering has broken for TNT Drama. Here is a potential new config:

```json
{
  "www.tntdrama.com": [
    {
      "apfCaptions": true,
      "apfCaptionsBottom": "50px",
      "apfCaptionsSelector": "tui-cc__container",
      "captionControlStyle": true,
      "displaySelector": "div.tui-cc__container",
      "displayVisibility": true,
      "mode": "watcher",
      "subtitleSelector": "div.tui-cc__container div.cc-renderer-cue"
    }
  ]
}
```

To use the config, paste it into the Custom Sites area (Options page > Audio tab) and click the blue "SAVE" button underneath it. Then, make sure to refresh the tab for the video site (if you have it open still) and it will load this new configuration.

**Comment by richardfrost** [comment]
This was included in version [4.3.0](https://github.com/FrostCo/AdvancedProfanityFilter/releases/tag/4.3.0). Feel free to remove the custom config if you added it so that you will automatically get updates for those sites in the future.
